packages/react/src/task-list/TaskListHeading.tsx

```diff
@@ -11,9 +11,9 @@ export const TaskListHeading = ({
 }) => {
 	return (
 		<Stack gap={1}>
-			<H2>Task list</H2>
+			<H2>Complete these tasks</H2>
 			<Text fontSize="sm" color="muted">
-				{stepsCompleted} of {totalSteps} steps completed
+				{stepsCompleted} of {totalSteps} tasks completed
```

Comment:
Is it worth updating all of the variables too?

Reply:
I considered that and actually thought this was a composable component so opted not to because it would be a prop name change. But it's not composable, so it's probably worthwhile.